libpyvinyl/Parameters/Parameter.py

```diff
@@ -1,34 +1,164 @@
 # Created by Mads Bertelsen and modified by Juncheng E
```

Comment:
I'd omit these attributions. git knows the authorship of every line in the code

Reply:
I think we should setup a commented header for each file with authors, the grant, etc.
Git knows but people might just copy the file.
What do you think?

Reply:
ok